website/src/theme/ReactLiveScope/components.tsx

```diff
@@ -1,6 +1,6 @@
-import React from 'react';
+import React, {HTMLAttributes} from 'react';
 
-export const ButtonExample = (props) => (
+export const ButtonExample = (props: HTMLAttributes<HTMLButtonElement>) => (
```

Comment:
Would also use `ComponentProps<'button'>` ;)

Reply:
Thanks for that 😅 Can't believe I spent five minutes figuring out why `ComponentProps<HTMLButtonElement>` is wrong